azurerm/resource_arm_healthcare_service_test.go

```diff
@@ -0,0 +1,133 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+)
+
+func TestAccAzureRMHealthcareService(t *testing.T) {
+	ri := tf.AccRandTimeInt()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMHealthcareServiceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMHealthcareService_basic(ri),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMHealthcareServiceExists("azurerm_healthcare_service.test"),
+				),
+			},
+			{
+				ResourceName:            "azurerm_healthcare_service.test",
+				ImportState:             true,
+				ImportStateVerifyIgnore: []string{
+					// since these are read from the existing state
```

Comment:
if this isn't needed could we remove the `ImportStateVerifyIgnore`

Reply:
done